### ISSUE TITLE
try to revive borg-linuxold64

### ISSUE THREAD

**Issue by ThomasWaldmann** [borgbackup/borg#7010]
looks like rhel7 & co does not work with the newer binaries.

so, check if the debian stretch vm can be revived for borg2.

**Comment by ThomasWaldmann** [comment]
https://github.com/borgbackup/borg/commit/fc67453bf3e17000da10ba3a4dfb691f39294633

**Comment by ThomasWaldmann** [comment]
I tried to build borg2 using the stretch64 vagrant VM.

We can't build borg2 (master branch) on debian stretch because **there is no `libxxhash-dev`** there (neither there is such a package in stretch-backports) and it is **required** for borg2.

So everybody needing a borg working on a rather old system and used the binary we built on debian stretch needs to stick to borg < 2.0 (== borg 1.2.x or 1.1.x) until they can upgrade the OS to something newer.

Should not be a problem: if you run that old systems, running a not brand new borg should be adequate.
